frappe/utils/backups.py

```diff
@@ -40,13 +40,13 @@ def __init__(
 		db_name,
 		user,
 		password,
+		db_host,
+		db_port,
+		db_type,
```

Comment:
Why make this args... kinda unnecessary breaking change? 

Remove bad defaults (use `None`) and use conf values instead. That will be consistent with previous behaviour.

Reply:
Also when args hit count of >3 it's hard to keep track of what's being passed where, better to stick with kwargs.